docs/contributing.md

```diff
@@ -1,5 +1,5 @@
 # How to contribute to Azure TRE
-> To read about how to set up your development environment and contribute to Azure TRE components please visit [Local Development](/using-tre/local-development/local-development/).
+> To read about how to set up your development environment and contribute to Azure TRE components please visit [Local Development](./using-tre/local-development/local-development.md).
```

Comment:
```suggestion
> To read about how to set up your development environment and contribute to Azure TRE components please visit [Local Development](../using-tre/local-development/local-development/).
```

Reply:
Fix is slightly different, I created a [PR](https://github.com/microsoft/AzureTRE/pull/3417)